test_opensearchpy/test_client/test_point_in_time.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+#  Licensed to Elasticsearch B.V. under one or more contributor
+#  license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright
+#  ownership. Elasticsearch B.V. licenses this file to you under
+#  the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+from test_opensearchpy.test_cases import OpenSearchTestCase
+
+
+class TestPointInTime(OpenSearchTestCase):
+    def test_create_one_point_in_time(self):
+        index_name = "test-index"
+        self.client.create_point_in_time(index=index_name)
+        self.assert_url_called("POST", "/test-index/_search/point_in_time")
+
+    def test_delete_one_point_in_time(self):
+        self.client.delete_point_in_time(body={})
```

Comment:
we should not call delete with empty body . its delete all or delete with list of pit ids in empty.

Reply:
Sure, have updated it in the new revision. Thanks!